### ISSUE TITLE
not working on iOS and Android, only web?

### ISSUE THREAD

**Issue by Chriko2502** [callstack/react-native-slider#527]
I wanted to try out the slider and used expo snack for this:
https://snack.expo.dev/@chr1k0/slider-example

Unfortunately it is only working in the web version. Not on Android or iOS.

Getting the error:
"View config getter callback for component `RNCSlider` must be a function (received `undefined`)"

Thanks in advance!

**Comment by BartoszKlonowski** [comment]
Hello @Chriko2502!
Unfortunately I couldn't reproduce the issue you reported.
Please check the PR to the repro app I linked to see if the scenario matches yours and if you are able to maybe see the issue there as well?
Also, it would help me to further analyze your issue if I'm provided with a repro repository that I can clone and work on.

**Comment by BartoszKlonowski** [comment]
Closing due to no reproduction possible and no author's response. Also I don't see other community folks having this issue.
Feel free to leave a comment if issue is noticed again.

**Comment by johnpill1** [comment]
I've encountered this issue too. 
<img width="1031" alt="Screenshot 2023-09-05 at 20 54 20" src="https://github.com/callstack/react-native-slider/assets/28736855/7ad8a7b4-953a-4572-b933-41600ddc5d29">
<img width="671" alt="Screenshot 2023-09-05 at 20 53 59" src="https://github.com/callstack/react-native-slider/assets/28736855/f999843e-d1dc-45fe-a3d5-e5a5d130937f">
<img width="252" alt="Screenshot 2023-09-05 at 20 54 26" src="https://github.com/callstack/react-native-slider/assets/28736855/70e09f57-70ff-4a2e-bd0a-383b7cdd6cd9">

**Comment by Ali-Aref** [comment]
+1 the same issue here. using expo and `"@react-native-community/slider": "4.4.2"`

**Comment by pmick** [comment]
Hey! Adding some context here that might people with this issue. I'm working on a fork of the repo and I was running into the same issue `"View config getter callback for component RNCSlider must be a function (received undefined)"`. I started getting this after running `npm install` to generate the `package/dist` folder. I was able to make this error stop happening by deleting `package/node_modules` after generating the `dist` folder. From looking in the hermes debugger in my local project it looked like this package was using modules from the `node_modules` folder nested in package rather than the node_modules for the projects that were importing this slider. This might be the same issue.
